tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,108 @@
+<tool id="cp319" name="Cellprofiler" version="@CP_VERSION@">
+    <description>Run a CellProfiler pipeline</description>
+    
+    <macros>
+        <import>macros.xml</import>
+        <xml name="test_assert_content" token_n="291">
+            <assert_contents>
+                <has_n_lines n="@N@" />
+            </assert_contents>
+        </xml>
+    </macros>
+
+    <expand macro="requirements" version="@CP_VERSION@" package="cellprofiler" />
+    <expand macro="stdio" />
+
+    <command><![CDATA[
+        bash '$script_file' &&
+        cellprofiler -c -r --file-list list.txt -o ./output -p '$pipeline' 
+    ]]></command>
+
+    <configfiles>
+        <configfile name="script_file">
+mkdir ./input
+mkdir ./output
+#for $i,$filename in enumerate($input_type.inputdir):
+    ln -s '$filename' ./input/${filename.element_identifier}.${filename.ext}
+#end for
+
+find `pwd`/input -name *.tiff > list.txt
+
+        </configfile>
+    </configfiles>
+  
+    <inputs>
+        <param name="pipeline" type="data" format="txt" label="Pipeline file"/>
+            <conditional name="input_type">
+                <param name="type" type="select" label="Select either a zip file or a collection from history">
+                    <option value="zip">zip file</option>
+                    <option value="images">images</option>
+                </param>
+                <when value="zip">
+                    <param name="inputdir" type="data" format="zip" label="Select image dataset from history" /> 
+                </when>
+                <when value="images">
+                    <param name="inputdir" type="data" multiple="true" format="tiff,tif" label="Images"/>
```

Comment:
I see also "tif" here... Is there a way to have all the allowed formats in a list?